src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnection.cs

```diff
@@ -157,7 +156,6 @@ public SqlConnection(string connectionString, SqlCredential credential) : this()
             // else
```

Comment:
Also, the code from line 128 to 152 do not require, cause of they already have checked inside the `ConnectionString` property.

Reply:
The validation from line 128 to 152 is to check the connection options from the pool group while ConnectionString property setter checks the connectionString in use. Same logic in netfx.